aviatrix/resource_aviatrix_smart_group.go

```diff
@@ -3,13 +3,18 @@ package aviatrix
 import (
 	"context"
 	"fmt"
+	"regexp"
 
 	"github.com/AviatrixSystems/terraform-provider-aviatrix/v3/goaviatrix"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
 )
 
+var (
+	k8sNameRe = regexp.MustCompile(`^[a-z0-9]([a-z0-9-]{0,61}[a-z0-9])?$`)
```

Comment:
Might be nice to have a comment like
// K8s resource names must adhere to rfc1123 naming conventions.

Reply:
Renamed the variable to mention the rfc and added a comment.